src/web/routes/guidance/index.js

```diff
@@ -1,3 +1,5 @@
 module.exports = {
-  ...require('./guidance')
+  ...require('./guidance'),
+  ...require('./pages'),
+  ...require('./get-page-meta-data')
```

Comment:
This exports every function from `./get-page-meta-data` - it might be better to only expose the functions that you require:

```
const { getPageMeta } = require('./get-page-meta-data')

module.exports {
  ...require('./guidance'),
  ...require('./pages'),
  getPageMeta
}
```

Reply:
Fixed.